src/textual/_on.py

```diff
@@ -13,47 +14,70 @@ class OnDecoratorError(Exception):
     """Errors related to the `on` decorator.
 
     Typically raised at import time as an early warning system.
-
     """
 
 
 def on(
-    message_type: type[Message], selector: str | None = None
+    message_type: type[Message], selector: str | None = None, **kwargs: str
 ) -> Callable[[DecoratedType], DecoratedType]:
-    """Decorator to declare method is a message handler.
+    """Decorator to declare that method is a message handler.
```

Comment:
```suggestion
    """Decorator to declare that the method is a message handler.
```

I know that was missing in the original...